### PR TITLE
OSASINFRA-3506: openstack: Move quota and liveness jobs

### PR DESCRIPTION
**Pull request by pierreprinetti**
also use pre-built images instead of building them at each run.

Implements [OSASINFRA-3506](https://issues.redhat.com//browse/OSASINFRA-3506)

### CODE REVIEW

**Comment by pierreprinetti** [comment]
/pj-rehearse periodic-ci-shiftstack-ci-main-quotas-vexxhost periodic-ci-shiftstack-ci-main-cleanup-vexxhost periodic-ci-shiftstack-ci-main-liveliness-mecha-central

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml [hunk 1]
```diff
@@ -39,53 +39,6 @@ tests:
   commands: find . -type f -name "*.sh" | xargs --verbose --no-run-if-empty shellcheck
   container:
     from: shiftstack-scripts-image
```
Comment (MaysaMacedo) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:41]: do we need to maintain this file if all the jobs are moved?

Comment (pierreprinetti) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:41]: Yes. This configuration file contains:
* the “promotion” stanza for building the container image containing the scripts
* the presubmit that runs shellcheck on the scripts

**Review by mandre — COMMENTED** [review]
/lgtm

**Pull request merged**